exercises/high-scores/.meta/generator/high_scores_case.rb

```diff
@@ -0,0 +1,11 @@
+require 'generator/exercise_case'
+
+class HighScoresCase < Generator::ExerciseCase
+  def workload
+    [
+      "scores = #{scores}",
+      "output = #{expected.inspect}",
```

Comment:
I'd prefer that the variable here was `expected`

Reply:
The reason why I went with `output` (also thought of `result`) was to make it easier to parse visually, since we have them in every spec:

```ruby
scores = [1, 2, 3, 4]
output = [1, 2, 3, 4]
result = [1, 2, 3, 4]
expected = [1, 2, 3, 4]
```

That being said, if you think the we're losing meaning by using those variable names, I'll change it. What do you think?

Reply:
I'd rather choose variable names because they are meaningful rather than because they make the code look prettier.

You can always add a space after "scores" if you really care about lining things up.

"output" can be ambiguious, what is it the "output" of? In this case I understand it to be the the `expected_output` which I'd rather shorten to `expected`
 as it makes the `assert_equal` line make more sense.

The general format being:
`assert_equal expected, actual`

Then we have:

`assert_equal output, HighScores.new(scores).#{property}`
vs
`assert_equal expected, HighScores.new(scores).#{property}`

"expected" is also consistent with what most of the other exercises use in their tests.

(Although, this might not be a valid argument since I'm probably responsible for all of them being that way :) )

Reply:
Just to be clear: I think you should change it.

Reply:
Thank you for the detailed comments - I don't feel strongly about this and I'm typically not a proponent of making sure everything lines up because in "real life" it tends to be hard to maintain. 

This alone is enough reason for me:

> "expected" is also consistent with what most of the other exercises use in their tests.

Consistency++ I'll change it!